src/content/posts/2024/09/03/tuist-4.26.0.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Tuist 4.26.0: GitHub integration and new command to find implicit dependencies"
+categories: ["Releases"]
+excerpt: "Learn more about the new features and improvements in Tuist 4.26.0."
+author: fortmarek
+---
+
+[Tuist 4.26.0](https://github.com/tuist/tuist/releases/tag/4.26.0) is out, and we'd like to share with you the new features and improvements that come with it.
+
+## Meeting developers where they are: GitHub integration
+
+One of the principles we embrace at Tuist is to meet developers where they are.
+
+- We chose Swift over [YAML](https://yaml.org/) for the manifest format because developers love writing in Swift. This choice also unlocks many possibilities that aren't achievable with YAML.
+- We integrated with and aligned ourselves with the [Swift Package Manager](https://www.swift.org/documentation/package-manager/) because it has become the standard for managing dependencies in the Swift ecosystem.
+- We added support for [binary caching](https://docs.tuist.io/guides/develop/build/cache) using Xcode project primitives instead of asking developers to change their build system.
+
+**But where exactly are Xcode developers?** While they spend a large portion of their time in Xcode, they also spend significant time on platforms like [GitHub](https://github.com) or [Slack](https://slack.com), collaborating with their peers. **Should we meet them there?** We think so!
+
+Traditionally, meeting developers on these platforms meant teams had to develop and maintain complex CI workflows that pushed data to these platforms. Although this approach worked, it exposed organizations to malicious actors because secrets were often revealed in CI workflows. Furthermore, the absence of a place to store data over time limited what teams could do with these integrations. **Imagine being notified when a pull request introduces a regression in build times** or **when a newly introduced test in the codebase is flaky.**
+
+At Tuist, we had the most crucial element to enable this type of developer experience: [a server](https://docs.tuist.io/server/introduction/why-a-server) that persists information from your projects and runs over time. However, we were missing one key piece—the integration of our server with one of the platforms where developers spend most of their time, [GitHub](https://github.com). That’s why we’re excited to announce that Tuist now integrates with GitHub. With a simple command, you can connect your remote project with a GitHub repository.
+
+```bash
+tuist project update tuist/tuist --repository-url https://github.com/tuist/tuist
+```
```

Comment:
You also need to install the GitHub app: https://github.com/marketplace/tuist

I'd also link to the [docs](https://docs.tuist.io/server/introduction/integrations#github).